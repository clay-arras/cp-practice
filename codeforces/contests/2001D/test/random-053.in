1
17
14 23 13 6 9 21 15 13 16 8 22 21 18 3 10 3 22
