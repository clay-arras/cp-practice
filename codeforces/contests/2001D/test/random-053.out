13
14 23 13 6 9 15 16 8 22 21 18 3 10
